index.html

```diff
@@ -31,10 +31,10 @@
 
         <div class="collapse navbar-collapse" id="navbarNavDropdown">
             <ul class="navbar-nav mr-auto">
-                <li class="active nav-item mr-2"><a class="nav-link" href="index.html">Home</a></li>
-                <li class="nav-item mr-2"><a class="nav-link" href="#">About</a></li>
-                <li class="nav-item mr-2"><a class="nav-link" href="pages/contact-page/contact.html">Contact Us</a></li>
-                <li class="nav-item mr-2"><a class="nav-link" href="pages/game-page/game.html">Play to Win</a></li>
+                <li class="active nav-item mr-2"><a class="nav-link" href="">Home</a></li>
+                <li class="nav-item mr-2"><a class="nav-link" href="../Bowes-Irish-whiskey-shop/pages/about-page/about.html">About</a></li>
+                <li class="nav-item mr-2"><a class="nav-link" href="../Bowes-Irish-whiskey-shop/pages/contact-page/contact.html">Contact Us</a></li>
+                <li class="nav-item mr-2"><a class="nav-link" href="../Bowes-Irish-whiskey-shop/pages/game-page/game.html">Play to Win</a></li>
                 <li class="nav-item mr-2 d-block d-sm-none"><button type="button" class="nav-link btn" data-toggle="modal" data-target="#signupmodal">Sign Up</button></li>
```

Comment:
Hey @reachjibi  I'm not 100% on how best to path through to the other pages, but I think contact.html from index.html for example might be href="/pages/contact-page/contact.html". Again though I could be wrong here.

Reply:
@phillydunne @reachjibi  yeah all three `<li>` paths need to be changed here

Reply:
href="pages/about-page/about.html">About</a>

href="pages/contact-page/contact.html">Contact Us</a>

href="pages/game-page/game.html">Play to Win</a>